plugins/modules/one_vm.py

```diff
@@ -76,9 +78,9 @@
     type: bool
   wait:
     description:
-      - Wait for the instance to reach its desired state before returning. Keep in mind if you are waiting for instance to be in running state
-        it does not mean that you will be able to SSH on that machine only that boot process have started on that instance, see 'wait_for' example
-        for details.
+      - Wait for the instance to reach its desired state before returning. Keep in mind if you are waiting for instance to
+        be in running state it does not mean that you will be able to SSH on that machine only that boot process have started
+        on that instance. See the example using the M(ansible.builtin.wait_for_instance) for details.
```

Comment:
```suggestion
        on that instance. See the example using the M(ansible.builtin.wait_for) for details.
```

Reply:
There is no example using `wait_for`. There is only one using `wait_for_instance`. So I guess the example needs to be fixed as well.

Reply:
I believe you mean this one:
```yaml
- name: Wait for SSH to come up
  ansible.builtin.wait_for_connection:
  delegate_to: '{{ vm.instances[0].networks[0].ip }}'
```
Which should be, in theory, if not in practice:
```yaml
- name: Wait for SSH to come up
  ansible.builtin.wait_for:
    port: 22  # it says SSH ^
    host: '{{ vm.instances[0].networks[0].ip }}'  # ??? not sure
```

Reply:
Yep, I meant that one.

Reply:
The right form is probably:
```yaml
- name: Wait for SSH to come up
  ansible.builtin.wait_for:
  delegate_to: '{{ vm.instances[0].networks[0].ip }}'
```

Reply:
I think I screwed up during copy'n'paste, I meant `ansible.builtin.wait_for_connection`, as it originally was :facepalm:
That's the module where `delegate_to` makes sense (because it's an action plugin, not a module).